### ISSUE TITLE
iOS build failing on clean project startup

### ISSUE THREAD

**Issue by bartolkaruza** [Microsoft/react-native-code-push#142]
I'm trying to get a react-native code push setup for iOS to work. I'm running into a compilation issue in XCode. I followed all the setup steps at the (getting started)[https://github.com/Microsoft/react-native-code-push#getting-started] on react-native init created project.

My JS dependencies:
`````
  "dependencies": {
    "react-native": "^0.18.0-rc",
    "react-native-code-push": "^1.5.2-beta"
  }
`````
In CodePush.m I'm getting the following compilation issue;

<img width="795" alt="screen shot 2016-01-09 at 3 23 29 pm" src="https://cloud.githubusercontent.com/assets/1605731/12216519/1d38084c-b6e5-11e5-9bfb-eeb6846d0dd7.png">

xcode: Version 7.1.1 (7B1005)
npm: 3.5.1
node: v5.0.0
OSX: 10.10.5 (14F27)

**Comment by falsecz** [comment]
duplicate https://github.com/Microsoft/react-native-code-push/issues/138

**Comment by lostintangent** [comment]
React Native 0.18.0 introduced a breaking change that caused this issue. We fixed it in master so if you want to grab the CodePush bits from this repo, it will work. I'm also planning on releasing an update to NPM to address this later this afternoon.

**Comment by lostintangent** [comment]
Just published 1.5.3 to NPM. Thanks for reporting this issue, and let us know if you hit any other problems.
